dts/bindings/dma/st,stm32-dma.yaml

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+title: STM32 DMA
+version: 0.1
+
+description: >
+    This is a representation of the STM32 DMA nodes
+
+inherits:
+  !include dma.yaml
+
+properties:
+    compatible:
+      constraint: "st,stm32-dma"
+
+    reg:
```

Comment:
by inherit from dma & base.yaml you can remove a lot of duplication here. (for things like reg).